tests/analysis/test_functional_groups.py

```diff
@@ -110,6 +111,7 @@ def test_get_all_functional_groups(self):
         assert len(all_func) == (len(link) + len(basics))
         assert sorted(all_func) == sorted(link + basics)
 
+    @pytest.mark.skipif(os.name == "nt", reason="Tests for openbabel failing on Win")
```

Comment:
i think `sys.platform == 'windows'` is more readable and [seems to be generally preferred](https://stackoverflow.com/a/8220141) to `os.name`

Reply:
Wonderful! Would do that.

I'm sure it's a typo, `sys.platform` should return `win32`, and `platform.system` would return `Windows`